gwpy/io/wav.py

```diff
@@ -0,0 +1,129 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) Duncan Macleod (2013)
+#
+# This file is part of GWpy.
+#
+# GWpy is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# GWpy is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with GWpy.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Write a `TimeSeries` to a WAV file
+
+"""
+
+from numpy import (savetxt, loadtxt)
+
+from scipy.io import wavfile
+
+
+from ..types import TimeSeries
+from .registry import (register_reader, register_writer, register_identifier)
+from .utils import identify_factory
+
+__author__ = "Ryan Fisher <ryan.fisher@ligo.org>"
+
+
+#def read_ascii(filepath, _obj=Series, xcol=0, ycol=1, delimiter=None,
```

Comment:
Left those comment blocks in to show the relative difference between wav and another io format.  This was to better enable you and Derek to review it.  We can remove the comments after you and Derek review the code if you prefer.